### ISSUE TITLE
RepresentativeAction fails on Symmetric and Alternating Groups

### ISSUE THREAD

**Issue by ChrisJefferson** [gap-system/gap#1431]
RepresentativeAction produces incorrect answers for both symmetric and alternating groups, with both `OnTuples` and `OnSets`, by producing elements outside the group. This bug was found by Mun See Chang

For example:

```
gap> RepresentativeAction(SymmetricGroup([5,7,11,15]),[7,11],[5,15],OnTuples);
(5,6,7)(11,15,14,13,12)
```

**Comment by ChrisJefferson** [comment]
Fixed in master in #1432, will need a seperate patch for 4.8.

**Comment by stevelinton** [comment]
PR #1453 should address this in stable-4.8.
